.circleci/config.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: artsy/hokusai
+    working_directory: ~/doppler
+    parallelism: 3
```

Comment:
we don't need this, lets just run it without parallel builds if its already fast enough